app/controllers/article_controller.rb

```diff
@@ -70,30 +70,43 @@ def eds_init
     config.show.abstract = 'eds_abstract'
     config.show.plink = 'eds_plink'
     config.show.route = { controller: 'article' }
-
-    # solr fields to be displayed in the show (single result) view
-    #   The ordering of the field names is the order of the display
-    config.add_show_field 'eds_title', label: 'Title'
-    config.add_show_field 'eds_composed_title', label: 'Composed Title'
-    config.add_show_field 'eds_other_titles', label: 'Other Titles'
-    config.add_show_field 'eds_languages', label: 'Language'
-    config.add_show_field 'eds_physical_description', label: 'Physical Description'
-    config.add_show_field 'eds_source_title', label: 'Journal'
-    config.add_show_field 'eds_database_name', label: 'Database'
-    config.add_show_field 'eds_authors', label: 'Author'
-    config.add_show_field 'eds_author_affiliations', label: 'Author Affiliations'
-    config.add_show_field 'eds_author_supplied_keywords', label: 'Author Supplied Keywords'
-    config.add_show_field 'eds_publication_type', label: 'Format'
-    config.add_show_field 'eds_document_type', label: 'Document Type'
-    config.add_show_field 'eds_publication_date', label: 'Publication Date'
-    config.add_show_field 'eds_publication_info', label: 'Published'
-    config.add_show_field 'eds_abstract', label: 'Abstract'
-    config.add_show_field 'eds_notes', label: 'Notes'
-    config.add_show_field 'eds_subjects', label: 'Subjects'
-    config.add_show_field 'eds_subjects_geographic', label: 'Geographic Subjects'
-    config.add_show_field 'eds_subjects_person', label: 'Person Subjects'
-    config.add_show_field 'eds_document_oclc', label: 'OCLC'
-    config.add_show_field 'eds_document_doi', label: 'DOI', helper_method: :doi_link
+    config.show.sections = {
+      'Summary' => {
+        eds_authors: 'Authors',
+        eds_author_affiliations: 'Author Affiliations',
+        eds_source_title: 'Journal',
+        eds_publication_date: 'Publication Date',
+        eds_languages: 'Language'
+      },
+      'Abstract' => {
+        eds_abstract: 'Abstract',
+        eds_notes: 'Notes'
+      },
+      'Subjects' => {
+        eds_subjects: 'Subjects',
+        eds_subjects_geographic: 'Geography',
+        eds_subjects_person: 'Person Subjects',
+        eds_author_supplied_keywords: 'Author Supplied Keywords'
+      },
+      'Details' => {
+        eds_publication_type: 'Format',
+        eds_document_doi: 'DOI',
+        eds_database_name: 'Database',
+        eds_composed_title: 'Composed Title',
+        eds_volume: 'Volume',
+        eds_series: 'Series',
+        eds_issue: 'Issue',
+        eds_page_start: 'Page Start',
+        eds_page_count: 'Page Count',
+        eds_isbns: 'ISBN',
+        eds_issns: 'ISSN',
+        eds_publication_info: 'Published',
+        eds_document_oclc: 'OCLC',
+        eds_document_type: 'Document Type',
+        eds_other_titles: 'Other Titles',
+        eds_physical_description: 'Physical Description'
```

Comment:
Thanks for the feedback. I think I've got something working now that keeps the `add_show_field` and feeds through the default presenter. The only concern I have is that rather than multiple values showing up on different lines, they are "and"ed together like so:

![screen shot 2017-07-06 at 2 56 02 pm](https://user-images.githubusercontent.com/1861171/27934593-4481c2c6-625b-11e7-8339-041b64ba4db4.png)

maybe there's a way to fix that in the rendering pipeline? Previously I was using the presenter to do a join with `'</dd><dd>'`...